OBSDOCS-1348: Add support exception statement in Logging docs for COO due to its Logging UI Plugin (#83306)
openshift/openshift-docs: observability/logging/logging-6.0/log6x-visual.adoc

toc::[]

> Visualization for logging is provided by deploying the xref:../../../observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc#logging-ui-plugin[Logging UI Plugin] of the xref:../../../observability/cluster_observability_operator/cluster-observability-operator-overview.adoc#cluster-observability-operator-overview[Cluster Observability Operator], which requires operator installation.

Reviewer: ```suggestion
The xref:../../../observability/cluster_observability_operator/cluster-observability-operator-overview.adoc#cluster-observability-operator-overview[Cluster Observability Operator] (COO) provides logging visualization by deploying its xref:../../../observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc#logging-ui-plugin[Logging UI Plugin], which requires operator installation.
```

Author: Thank you for the suggestion. I think the original wording here was chosen in order to put emphasis on the visualization feature and because this doc is about Logging rather than the COO. So I'd prefer to not change the original wording.

> Visualization for logging is provided by deploying the xref:../../../observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc#logging-ui-plugin[Logging UI Plugin] of the xref:../../../observability/cluster_observability_operator/cluster-observability-operator-overview.adoc#cluster-observability-operator-overview[Cluster Observability Operator], which requires operator installation.

Reviewer 2: Capitalize "Operator":
```suggestion
Visualization for logging is provided by deploying the xref:../../../observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc#logging-ui-plugin[Logging UI Plugin] of the xref:../../../observability/cluster_observability_operator/cluster-observability-operator-overview.adoc#cluster-observability-operator-overview[Cluster Observability Operator], which requires Operator installation.
```

Author: Thank you. Fixed.
